libs/common/src/lib/runtime-track.ts

```diff
@@ -52,6 +53,7 @@ export type RuntimeTrack = {
   // maximum distance between two consecutive points.
   maxDistance: number;
   airspaces?: protos.Airspaces;
+  score?: Score;
```

Comment:
I don't think there is a need to store the score in the track for now.
Maybe at some point it could be helpful but probably not needed in the first iteration.

Reply:
removed